### DISCUSSION TITLE
Coregister and segmentation

### DISCUSSION THREAD

**Discussion by HolgeMax** [community/community#110622] [Programming Help]
### Select Topic Area

Question

### Body

Hallo discussion,

I am trying to make a mask from GW, WM and CSF using segmentation and the image calculator. I have encountered a problem with the orientation and rotation of the t1w and coreg t2w images. After doing coregister, estimate the transformation matrix for the t2w image. Then, by using dependency, I am trying to do a multi-spectral segmentation with t1w channel 1 and coreg t2 channel 2. I thought it would work since the coregistration for t1w on t2w should be a purely linear transformation, and the transformation information would be saved in the header and applied using t1w and coreg t2w dependency. But this does not work as the images still have different orientationsC€
![Batch](https://github.com/community/community/assets/143646321/6cbba7ce-e45b-4c9b-9b2f-1d5cfe68305b)
![Coreg](https://github.com/community/community/assets/143646321/40139e6b-e246-42cc-976f-6e0f0c8300ca)
![Error message](https://github.com/community/community/assets/143646321/768d36f3-a849-4ab1-9c1e-b7d4825295ae)
![Segmentation](https://github.com/community/community/assets/143646321/72e8bbb8-2832-4604-93b7-773d2e33c058)
 and rotations, so the segmentation fails:

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @HolgeMax ! 

We’ve moved your post to our Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion.

Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information.
